lib/identity_doc_auth/mock/result_response_builder.rb

```diff
@@ -43,11 +45,28 @@ def call
       private
 
       def errors
-        error = parsed_data_from_uploaded_file&.dig('friendly_error')
-        if error.blank?
+        file_data = parsed_data_from_uploaded_file
+
+        if file_data.blank?
           {}
         else
-          { results: [error] }
+          image_metrics = file_data&.dig('image_metrics') || {}
+          failed = file_data&.dig('failed_alerts') || []
+          passed = file_data&.dig('passed_alerts') || []
+          liveness_result = file_data&.dig('liveness_result') || ''
+
+          if [image_metrics,failed, passed, liveness_result].any?(&:present?)
+            fake_response_info = create_response_info(
+              image_metrics: image_metrics&.symbolize_keys,
+              failed: failed.map!(&:symbolize_keys),
+              passed: passed.map!(&:symbolize_keys),
```

Comment:
We could symbolize earlier in the YAML parsing? i.e. `YAML.safe_load(uploaded_file, symbolize_names: true)`

https://ruby-doc.org/stdlib-2.6.1/libdoc/psych/rdoc/Psych.html#method-c-load

Saves us the work here, and produces a more consistent result between `parse_uri` and `parse_yaml` fallbacks.

Reply:
I'll have to look. We have to be a little careful because some of this objects in the real responses aren't symbolized, because we've not consistent about symbols vs strings in hashes, especially when we're not significantly modifying the object (which happens for image metrics).

Reply:
yeah, I'd need to fix the real implementation of image metrics in both acuant and trueid, which is a little more than I'd like to do for this update to our mocking functionality. So, I'll leave a note and fix that with some of the other upcoming work in this area.